install-mac.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+brew bundle --file=- <<-EOS
```

Comment:
Why not use `Brewfile`?

Reply:
dont remember why i was doing this via brewfile anymore. left if as normal brew install commands. Anything you can remember that would make us needed the brewfile?